tests/utils/test_validation.py

```diff
@@ -69,6 +74,9 @@
     "a" * 257,
     None,
     "$dgs",
+    "latest",
```

Comment:
```suggestion
    "Latest",
    "latest",
```

to ensure case insensitivity